docs/2_4_common_schema.adoc

```diff
@@ -1077,12 +1077,19 @@ The default value of this attribute is `false`.
 |
 [[previous,`previous`]]
 If present, this variable is a <<ClockedState>> and this attribute is a value reference to the variable with the previous value.
-Only clocked variables (they are listed in at least one of the <<clockedVariables>> lists) may have the <<previous>> attribute.
+Only clocked variables (i.e. variables with the attribute <<clocks>>) may have the <<previous>> attribute.
 Only variables with <<variability,`variability == discrete`>> may have a previous value.
 
 _[For example, if `previous == 3` for variable `8`, then variable `3` is the previous value of variable `8`. See also <<fmi3UpdateDiscreteStates>>._
 _Note: This is reverse compared to the <<derivative>> attribute.]_
 
+|`clocks`
+|
+[[clocks,`clocks`]]
+If present, this variable is clocked.
+The value of <<clocks>> is a list of value references of <<Clock>> variables this variable depends on.
+Only discrete variables can have this attribute.
```

Comment:
> Only discrete variables can have this attribute.

We need the information also for tunable parameters, i.e. discrete variables and tunable parameters can have this attribute.